tests/NPSP.robot

```diff
@@ -280,5 +290,17 @@ Select Frame With Title
     
 Scroll Page To Location
     [Arguments]    ${x_location}    ${y_location}
-    Execute JavaScript    window.scrollTo(${x_location},${y_location}) 
-    
\ No newline at end of file
+    Execute JavaScript    window.scrollTo(${x_location},${y_location})
+
+Open NPSP Settings
+    [Arguments]    ${topmenu}    ${submenu}
+    Open App Launcher
+    Populate Address    Search apps or items...    NPSP Settings
+    Select App Launcher Link  NPSP Settings
```

Comment:
I think these 3 lines can be replaced by `Select App Launcher Tab  NPSP Settings` (grr, github won't let me enter two consecutive spaces between the keyword and the argument there)

Reply:
Didn't realize there was a command for that. Will give it a try.